ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/interfaces/AbstractResponse.java

```diff
@@ -113,6 +113,15 @@ public void checkStatusCode(final int ... expectedStatuses) throws IOException {
         }
     }
 
+    private boolean checkForManagementPortException() {
+        if (this.getStatusCode() == FailedRequestUsingMgmtPortException.MGMT_PORT_STATUS_CODE) {
```

Comment:
This can be consolidated into a single boolean expression.  And the result of that boolean expression and be returned.  There is no need to return true or false explicitly.